x-pack/plugins/monitoring/public/components/summary_status/index.js

```diff
@@ -4,4 +4,73 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export { SummaryStatus } from './summary_status';
```

Comment:
Was this intentional? This looks the same as `summary_status/summary_status.js`

Reply:
This might also help fix one of the test failures:
```
23:46:50 /var/lib/jenkins/workspace/elastic+kibana+pull-request+multijob-intake/kibana/x-pack/plugins/monitoring/public/components/summary_status/index.js
23:46:50   11:10  error  StatusIcon not found in '../'  import/named
```